bigbluebutton-html5/imports/ui/components/audio/device-selector/component.jsx

```diff
@@ -129,13 +129,13 @@ class DeviceSelector extends Component {
     } = this.props;
 
     const { options } = this.state;
-    const { isSafari } = browserInfo;
+    const { isSafari, isFirefox } = browserInfo;
 
     let notFoundOption;
 
     if (blocked) {
       notFoundOption = <option value="finding">{intl.formatMessage(intlMessages.findingDevicesLabel)}</option>;
-    } else if (kind === 'audiooutput' && isSafari) {
+    } else if (kind === 'audiooutput' && (isSafari || isFirefox)) {
```

Comment:
would it make sense to drop the isSafari and isFirefox tests completely? I tested the patch with Firefox setting `media.setsinkid.enabled` set to true and it shows the outputs because `options` are filled by the browser callbacks as soon as devices are discovered.

```suggestion
    } else if (kind === 'audiooutput' && !('setSinkId' in HTMLMediaElement.prototype)) {
```

Reply:
Tested with Chrome, Firefox, Edge dev on Linux, Safari on OSX. Seems to work